common/persistence/nosql/nosqlplugin/cassandra/gocql/client.go

```diff
@@ -51,7 +51,10 @@ func RegisterClient(c Client) {
 func newCassandraCluster(cfg ClusterConfig) *gocql.ClusterConfig {
 	hosts := parseHosts(cfg.Hosts)
 	cluster := gocql.NewCluster(hosts...)
-	cluster.ProtoVersion = 4
+	if cfg.ProtoVersion == 0 {
+		cfg.ProtoVersion = 4
```

Comment:
Since you have declare the global constant, this cone should become
```
cfg.ProtoVersion = env.CassandraDefaultProtoVersion
```

Reply:
What about using the function `environment.GetCassandraProtoVersion()` that I created instead? (to avoid the int parsing). And another question would be, if the answer is yes, should I use it everywhere instead of a const? (i.e `common/persistence/nosql/nosqlplugin/cassandra/session.go:39` in this diff)

Reply:
Instead of parsing, how about create another `environment.CassandraDefaultProtoVersionInteger` beside `environment.CassandraDefaultProtoVersion`? and always use environment.CassandraDefaultProtoVersionInteger to avoid parsing string.